src/components/reader-navigation.js

```diff
@@ -15,11 +15,17 @@ type Props = {
 
 const getChapterLabel = (chapter): string => {
   const parts = [
-    chapter.volumeNumber && `Vol. ${chapter.volumeNumber}`,
+    chapter.volumeNumber &&
+      chapter.volumeNumber !== '0' &&
```

Comment:
Treating `"0"` as a non-value might make more sense to do in [poketo/node](https://github.com/poketo/node) in the future.